AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -0,0 +1,74 @@
+# options
```

Comment:
To get these config files to appear in `Bundle.main` for the executable target, I had to move them from the `resources` folder to the `AirbnbSwiftFormatTool` plugin